### PR TITLE
Update mongodb documentation

### PR DESCRIPTION
**Pull request by perekalov**

### CODE REVIEW

File: MongoDB.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ## WAL-G for MongoDB
 
-**Interface of Mongo now is unstable**
+**Interface of MongoDB features now is unstable**
```
Comment (kuzpactor) [inline MongoDB.md:3]: `is currently unstable`.
"Now" here reads as "becomes"

File: MongoDB.md [hunk 2]
```diff
@@ -32,53 +32,72 @@ make mongo_install
...
+Format: [golang duration string](https://golang.org/pkg/time/#ParseDuration).
+
+
+* ##### `OPLOG_PUSH_WAIT_FOR_BECOME_PRIMARY`
+
+Wait for primary and start archiving or exit immediately. 
+
+* ##### `OPLOG_PITR_DISCOVERY_INTERVAL`
+
+Defines longest point-int-time recovery period for `oplog-purge`.
```
Comment (kuzpactor) [inline MongoDB.md:70]: `point-in-time`

File: MongoDB.md [hunk 3]
```diff
@@ -32,53 +32,72 @@ make mongo_install
...
+
+Interval to update the latest majority optime. wal-g archives only majority committed operations.
+Format: [golang duration string](https://golang.org/pkg/time/#ParseDuration).
+
+
+* ##### `OPLOG_PUSH_WAIT_FOR_BECOME_PRIMARY`
+
+Wait for primary and start archiving or exit immediately. 
+
+* ##### `OPLOG_PITR_DISCOVERY_INTERVAL`
```
Comment (kuzpactor) [inline MongoDB.md:68]: Perhaps, a more in-depth description of this option may be necessary.
May be I miss something, but I did not get the meaning of this key.

File: MongoDB.md [hunk 4]
```diff
@@ -131,7 +179,9 @@ Archiving collects writes if optime is readable by majority reads. Optime is upd
 wal-g oplog-push
 ```
 
-* ``oplog-replay``
+Note: archiving works only on primary, but you can run it on all replicaset nodes with config option `OPLOG_PUSH_WAIT_FOR_BECOME_PRIMARY: true`.
```
Comment (kuzpactor) [inline MongoDB.md:182]: `... on any replicaset node using config ...`

File: MongoDB.md [hunk 5]
```diff
@@ -154,3 +210,81 @@ Supported formats to output: `json`, `bson`, `bson-raw`
...
+### Full backup/restore only
+
+Here's typical wal-g configuration for that case:
+```
+MONGODB_URI:                 'mongodb://user:password@localhost:27018/?authSource=admin&connect=direct&socketTimeoutMS=60000&connectTimeoutMS=10000&tls=true'
+WALG_STREAM_CREATE_COMMAND:  'mongodump --archive --oplog --uri="mongodb://user:password@localhost:27018/?authSource=admin&connectTimeoutMS=10000&tls=true"'
+WALG_STREAM_RESTORE_COMMAND: 'mongorestore --archive --oplogReplay --uri="mongodb://user:password@localhost:27018/?authSource=admin&connectTimeoutMS=10000&tls=true"'
+```
+
+You may add ```--drop``` option to restore command, but do it with caution.
```
Comment (kuzpactor) [inline MongoDB.md:241]: A reminder of what `--drop` actually does would be nice here.

```
You may also add --drop option to restore command. This option will delete local files (?) and thus extra care should be exercised.
```

File: MongoDB.md [hunk 6]
```diff
@@ -154,3 +210,81 @@ Supported formats to output: `json`, `bson`, `bson-raw`
...
+OPLOG_ARCHIVE_TIMEOUT_INTERVAL: '30s'
+OPLOG_ARCHIVE_AFTER_SIZE:       '20971520'
+
+# Oplog-purge setting: retain oplog archives uploaded after oldest backup created during last 7 days
+OPLOG_PITR_DISCOVERY_INTERVAL: '168h'
+
+# Run and wait for become primary (usefull to run on all secondaries)
+OPLOG_PUSH_WAIT_FOR_BECOME_PRIMARY: 'true'
+
+# Enable archiving statistics collection
```
Comment (kuzpactor) [inline MongoDB.md:260]: Perhaps a more unambiguous --
`Collect statistics on archiving process`.

**Review by kuzpactor — CHANGES_REQUESTED** [review]

**Review by VGoshev — APPROVED** [review]

**Pull request merged**